### PR TITLE
fix PostgreSQL LSO

### PR DESCRIPTION
**Pull request by ebattat**
This PR should fix PostgreSQL LSO hammerdb issue.
The current results of ODF are better than LSO because lack of load transactions.

There are 2 changes in this PR:
1. Update load transactions to 200K instead of 100K
2. Add LSO verification in case of empty path.

### CODE REVIEW

File: benchmark_runner/benchmark_operator/benchmark_operator_workloads_operations.py [hunk 1]
```diff
@@ -455,6 +456,15 @@ def odf_pvc_verification(self):
             if not self._oc.is_odf_installed():
                 raise ODFNonInstalled()
 
+    @logger_time_stamp
+    def lso_verification(self):
+        """
+        This method verify that lso path exist
```
Comment (RobertKrawitz) [inline benchmark_runner/benchmark_operator/benchmark_operator_workloads_operations.py:462]: Verifies

File: benchmark_runner/benchmark_operator/benchmark_operator_workloads_operations.py [hunk 2]
```diff
@@ -455,6 +456,15 @@ def odf_pvc_verification(self):
             if not self._oc.is_odf_installed():
                 raise ODFNonInstalled()
 
+    @logger_time_stamp
+    def lso_verification(self):
```
Comment (RobertKrawitz) [inline benchmark_runner/benchmark_operator/benchmark_operator_workloads_operations.py:460]: "verify_lso" would be a better name.

File: benchmark_runner/common/template_operations/templates/hammerdb/hammerdb_data_template.yaml [hunk 3]
```diff
@@ -8,7 +8,6 @@ template_data:
   shared:
     pin_node1: {{ pin_node1 }}
     pin_node2: {{ pin_node2 }}
-    transactions: 100000
```
Comment (RobertKrawitz) [inline benchmark_runner/common/template_operations/templates/hammerdb/hammerdb_data_template.yaml:11]: You should be able to leave this here, and override it only for postgres.  You can also put the rampup time here.

**Review by RobertKrawitz — CHANGES_REQUESTED** [review]

File: benchmark_runner/benchmark_operator/benchmark_operator_workloads_operations.py (hunk 2 above)
Comment (ebattat) [reply benchmark_runner/benchmark_operator/benchmark_operator_workloads_operations.py:460]: Fixed

File: benchmark_runner/common/template_operations/templates/hammerdb/hammerdb_data_template.yaml (hunk 3 above)
Comment (ebattat) [reply benchmark_runner/common/template_operations/templates/hammerdb/hammerdb_data_template.yaml:11]: Fixed

**Review by RobertKrawitz — APPROVED** [review]
Noted in private discussion that the change of runtime from 2 to 1 is intentional.  Approving.

**Pull request merged**